ui/cypress/e2e/dataset_publish.e2e.cy.js

```diff
@@ -60,96 +60,103 @@ const genTransformPairs = (dataset_id) => {
 
 import { waitForElwood } from '../support/helpers';
 
+const username = Cypress.env('DOJO_DEMO_USER');
+const password = Cypress.env('DOJO_DEMO_PASS');
+const hasAuth = Boolean(username) && Boolean(password);
+const auth = hasAuth ? {username, password} : undefined;
+
+const isAuthEnabledRemote = hasAuth;
+
 
 describe('Dataset Register: Publish E2E', { browser: ['chrome', 'chromium', 'firefox'] }, () => {
 
   let dataset_id;
 
   afterEach(() => {
-    console.log('Cleaning seeded artifacts.');
-    cy.task('seed:clean', {type: 'dataset', id: dataset_id});
-    cy.task('seed:clean', {type: 'annotation', id: dataset_id});
-
+    if (!isAuthEnabledRemote) {
+      cy.log('Cleaning seeded artifacts.');
+      cy.task('seed:clean', {type: 'dataset', id: dataset_id});
+      cy.task('seed:clean', {type: 'annotation', id: dataset_id});
+    } else {
+      cy.log('Auth enabled, skip seed cleaning.');
+    }
     // Call seed:clean-files for minio files? TBD.
   });
 
   it('From Annotate > skip transform > Process/publish modifies the dataset correctly', () => {
 
     const dataset = genDataset('acled');
 
-    cy.request('POST', '/api/dojo/indicators', dataset)
-      .then(({body}) => {
+    cy.seed({type: 'dataset', data: dataset})
+      .then(({ body }) => {
+
+        console.log('created seed dataset');
 
         const createdDataset = body;
         dataset_id = createdDataset.id;
         const transformPairs = genTransformPairs(dataset_id);
 
-        cy.request('POST', `/api/dojo/indicators/${dataset_id}/annotations`, dataset_acled_annotations);
-        cy.task('upload', {type: 'dataset', id: dataset_id, variant: 'acled'});
+        cy.seed({type: 'annotation', data: dataset_acled_annotations, id: dataset_id})
```

Comment:
Main change on most test files is to use the `seed` command instead of making manual "seed" requests inline in tests.